packages/rspack-test-tools/tests/configCases/hooks/update-asset-patch-asset-info/rspack.config.js

```diff
@@ -0,0 +1,40 @@
+/** @type {import("@rspack/core").Configuration} */
+module.exports = {
+	output: {
+		filename: "[name].[contenthash].js"
+	},
+	plugins: [
+		function plugin(compiler) {
+			compiler.hooks.compilation.tap("test", compilation => {
+				compilation.hooks.processAssets.tap(
+					{
+						name: "test",
+						stage: compiler.webpack.Compilation.PROCESS_ASSETS_STAGE_ADDITIONS
+					},
+					assets => {
+						Object.entries(assets).forEach(([filename, asset]) => {
+							const newContent = `// UPDATED\n${asset.source()}`;
+							compilation.updateAsset(
+								filename,
+								new compiler.webpack.sources.RawSource(newContent),
+								{ minimized: true }
+							);
+						});
+					}
+				);
+				compilation.hooks.processAssets.tap(
+					{
+						name: "test",
+						stage:
+							compiler.webpack.Compilation.PROCESS_ASSETS_STAGE_OPTIMIZE_HASH
+					},
+					assets => {
+						compilation.getAssets().forEach(({ info }) => {
+							expect(info.contenthash.length).toBeGreaterThan(0);
```

Comment:
How does this case compare contenthash with the previous one?

Reply:
before this PR, here is undefined, the `asset_info.contenthash` is overridden by new asset info `compilation.updateAsset(..., { minimized: true })`, here is only ensure that `asset_info.contenthash` is not overridden unexpectedly